views/slackbot/slackbot.go

```diff
@@ -205,7 +206,17 @@ func sendCouchbaseCloudReplies(client *slack.Client, channelId string, couchbase
 		var message bytes.Buffer
 		message.WriteString(fmt.Sprintf("*Name*: `%s`\n", cloud.Name))
 		message.WriteString(fmt.Sprintf("*Provider*: `%s`\n", cloud.Provider))
-		message.WriteString(fmt.Sprintf("*Region*: `%s`\n", cloud.Region))
+		regions := "*Regions*: "
+		if cloud.CloudRegion.AwsRegion != "" {
+			regions += fmt.Sprintf("`AWS: %s`", cloud.CloudRegion.AwsRegion)
+			if cloud.CloudRegion.AzureRegion != "" {
+				regions += ", "
+			}
+		}
+		if cloud.CloudRegion.AzureRegion != "" {
+			regions += fmt.Sprintf("`Azure: %s`", cloud.CloudRegion.AzureRegion)
+		}
+		message.WriteString(regions)
```

Comment:
I think there's a bug here

![image](https://user-images.githubusercontent.com/19732909/152155401-0db01ca1-3f88-416d-9d4d-a8c8de94916c.png)

To make all this simpler we can probably just define a string e.g.

`"AWS: [%s], Azure: [%s]" `

And then fill in the placeholders one by one with `fmt.Sprintf`. This way an if check wouldn't be needed either, yes you'll get `Azure: []` when there aren't any Azure regions but it still looks ok 🙂